### DISCUSSION TITLE
'Share to Daily Dev' button

### DISCUSSION THREAD

**Discussion by syakirurahman** [dailydotdev/daily#1291] [Feature Requests]
We know that big social media like X (twitter), facebook, linkedin has their own share button where people can embed it in their blog.
Daily dev has a nice Community pick feature where user can suggest a high quality post. It will be better if a user that just read a helpful post anywhere in the internet provided by a button that link to Community pick.

**Comment by idoshamun** [comment]
Hi @syakirurahman,

That's a great idea. Thanks :)
